README/ReleaseNotes/v622/index.md

```diff
@@ -287,6 +287,31 @@ Even more includes will be "hidden" when ROOT configured with `-Ddev=ON` build o
 In that case ROOT uses `#ifdef R__LESS_INCLUDES` to replace unused includes by class forward declarations.
 Such `dev` builds can be used to verify that ROOT-based code really includes all necessary ROOT headers.
 
+### Multi-Python PyROOT
+
+In 6.22, the new (experimental) PyROOT is built by default. In order to build with the old PyROOT instead, the option
+`-Dpyroot_legacy=ON` can be used.
```

Comment:
Do we have this option already in 6.22?

Reply:
Hehe I was expecting this comment :smile: 

No, we don't have the option at the moment, but we will before we release: `pyroot_experimental` needs to disappear and we need to provide another option to build the old PyROOT. The name (`pyroot_legacy`) is a suggestion from my side, but if you have any other proposal let me know.